src/tests/end_to_end/end_to_end.py

```diff
@@ -0,0 +1,248 @@
+import json
+import os
+import time
+import uuid
+from pathlib import Path
+
+from google.cloud import bigquery, storage
+from googleapiclient.discovery import build
+from jinja2 import Environment, FileSystemLoader
+from test_utils import parse_config
+
+
+# Load integration test ddl/sql/data config file in DMT gcs config bucket
+def load_test_config(config_file=None, config=None, uniq_id: str = "", type=None):
+    env = Environment(loader=FileSystemLoader("src/tests/end_to_end/config"))
+    config_template = env.get_template(config_file)
+    print(config_file)
```

Comment:
Ideally we should use logging statements instead of prints for all the informational messages in this file.

Reply:
Updated code. Integration test logs can be viewed under log explorer, user can search log with log name filter starting with "integration-test-" followed by unique id taken in all the config (and prepared using cloud build trigger id)